contracts/child/ChildToken/ChildERC1155.sol

```diff
@@ -3,11 +3,12 @@ pragma solidity "0.6.6";
 import {ERC1155} from "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
 import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
 import {IChildToken} from "./IChildToken.sol";
+import {Initializable} from "../../common/Initializable.sol";
 
 // import {NetworkAgnostic} from "../../common/NetworkAgnostic.sol";
 
 // TODO: network agnostic
-contract ChildERC1155 is ERC1155, IChildToken, AccessControl {
+contract ChildERC1155 is ERC1155, IChildToken, AccessControl, Initializable {
```

Comment:
Remove `Initializable` here, all child tokens are not using proxy.

Reply:
yes will remove them

Reply:
done